doc/psidk/src/services/cpp-service/reference/system.md

```diff
@@ -0,0 +1,32 @@
+# System Functions
+
+Some parts of the C++ standard library that interact with the host system are restricted or unavailable in services.
+
+## I/O
+
+Only stdout and stderr are available. They both write to a single merged stream which is in included the transaction trace. stdin is not supported.
+
+Supported APIs:
+- `write`
+- `printf` family
+- `std::cout`, `std::cerr`, `std::clog`
+
+## Clocks
+
+UTC, monotonic, and CPU clocks are available to subjective services. The monotonic clock is only guaranteed to be monotonic within a block. The CPU clock measures CPU time spent on the current transaction.
```

Comment:
This should define or link to something that defines "subjective services"

Reply:
I can't find any existing documentation for subjective services and it certainly doesn't belong here. Maybe under services/concepts?

Reply:
Perhaps just in [services](https://github.com/gofractally/psibase/blob/main/doc/psidk/src/services.md) itself

Reply:
I think it should be deferred to another PR. Documentation of subjective services is only incidentally related and this PR is already quite large.